lib/factory_trace.rb

```diff
@@ -48,9 +49,6 @@ def start
     def stop
       return unless configuration.enabled
 
-      # This is required to exclude parent traits from +defined_traits+
-      FactoryBot.reload
```

Comment:
**Question:** Hello, could you please confirm that we don't need it? Why do we remote it? 

I don't recall all the details but I think it was here fore the described reason and I'm not sure if we are okay to drop it.

Reply:
I deleted it because the enum has a lazy evaluation, so after calling `FactoryBot.reload` we lose information about the traits created based on the enum.

But also, I would like to find the reason why it has been added and add a correspondent test to prove it.

Reply:
It could be we need better integration test to prove that.